filesystem.toml

```diff
@@ -175,6 +175,32 @@ Welcome to Redox OS!
 """
 
 [[files]]
-path = "/usr"
-data = "/"
+path = "/usr/bin"
+data = "/bin"
 symlink = true
+
+[[files]]
+path = "/usr/sbin"
+data = "/sbin"
+symlink = true
+
+[[files]]
+path = "/usr/var"
```

Comment:
Do some systems have a `/usr/var`? Archlinux doesn't seem to have such a directory.

Reply:
No, I don't think they do come to think of it. For some reason I thought Linux did, but, just verified it doesn't. I'll remove that. Please reject this PR and let me re-work it and I'll submit a new PR Shortly.